research/format.markdown

```diff
@@ -0,0 +1,138 @@
+## Draft 2 to Draft 3:
```

Comment:
This is a level 2 heading. A document should always also have one and only one level 1 heading that is the title of the doc. Your title can say what this document is even about and potentially a one paragraph description, otherwise other people might not understand what this document is even about

Reply:
Also no `:` in titles

Reply:
> This is a level 2 heading. A document should always also have one and only one level 1 heading that is the title of the doc. Your title can say what this document is even about and potentially a one paragraph description, otherwise other people might not understand what this document is even about

That's correct. Heading is very important